### ISSUE TITLE
TensorFlow Lite Object Detection - Plot training and validation losses

### ISSUE THREAD

**Issue by HripsimeS** [tensorflow/tensorflow#60116]
Hello. I am using the following notebook to train my dataset with efficientdet-lite0 model.
https://www.tensorflow.org/lite/models/modify/model_maker/object_detection

I see for each epoch we get the information of training loss "loss" and validation loss "val_loss". I would like to plot them on the same graph if possible, but as 'ObjectDetector' object has no attribute 'history' I can't do it in a classic way **model.history['loss']**
For TensorFlow Lite the model has been build and fit with **object_detector.create()**

Can you help me to figure out how to plot training and validation losses on the same graph in this case. Thanks in advance!

**Comment by tilakrayal** [comment]
@HripsimeS,
Looks like this is a duplicate of the issue [#60087](https://github.com/tensorflow/tensorflow/issues/60087). Could you please close this issue, since it is already being tracked there? Thank you!
